cmd/ltctl/main.go

```diff
@@ -152,15 +152,19 @@ func main() {
 	rootCmd.AddCommand(loadtestCmd)
 
 	sshCmd := &cobra.Command{
-		Use:     "ssh [instance]",
+		Use:     "ssh [instance] [command]",
```

Comment:
I think it would be better if we did this via flag, as this is optional. Like `ssh appserver --run <remote command>`. I'm not going to ask for it explicitly since using same handler for both `ssh` and `ssh list` is not ideal as well.

Reply:
Ok